pkgs/applications/science/logic/proverif/default.nix

```diff
@@ -16,6 +16,8 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin
     cp ./proverif      $out/bin
     cp ./proveriftotex $out/bin
+    mkdir -p $out/share/emacs/site-lisp
+    cp emacs/proverif.el $out/share/emacs/site-lisp/
```

Comment:
```suggestion
    install -D emacs/proverif.el $out/share/emacs/site-lisp/
```
I think so

Reply:
Thanks for the suggestion. I had to read the man page of `install` to get something correct though…